src/Repository/FailedLoginAttemptRepository.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\FailedLoginAttempt;
+use Doctrine\ORM\EntityRepository;
+
+class FailedLoginAttemptRepository extends EntityRepository
+{
+    const MAX_ATTEMPTS = 5;
+    const LAST_MINUTE = 'PT1M';
+    const LAST_TEN_MINUTES = 'PT10M';
+
+    public function save(FailedLoginAttempt $failedLoginAttempt): void
+    {
+        $this->_em->persist($failedLoginAttempt);
+        $this->_em->flush();
+    }
+
+    public function canLogin(string $login): bool
+    {
+        if (0 === $this->countAttempts($login, self::LAST_MINUTE)) {
```

Comment:
this test looks useless.

The last line of the method is already doing that job if I am not mistaken.

Reply:
No this check if you don't have attempts in the last minute. The other is for 10 minutes

Reply:
Thanks ;-)